tests/system/action/user/test_create.py

```diff
@@ -199,3 +208,320 @@ def test_create_empty_username(self) -> None:
+        self.set_management_level(
+            OrganisationManagementLevel.CAN_MANAGE_USERS, self.user_id
+        )
+        self.set_user_groups(self.user_id, [3, 6])
+        self.set_group_permissions(3, [Permissions.User.CAN_MANAGE])
+
+        response = self.request(
+            "user.create",
+            {
+                "username": "username",
+                "number_$": {"1": "number1", "4": "number1 in 4"},
+            },
+        )
+        self.assert_status_code(response, 403)
+        self.assertIn(
+            "You are not allowed to perform action user.create. Missing permissions {'_User.CAN_MANAGE for meeting 4'}",
+            response.json["message"],
+        )
+
+    def test_create_permission_OML_not_high_enough(self) -> None:
+        """ May create group A fields only """
+        self.permission_setup()
+        self.set_management_level(
+            OrganisationManagementLevel.CAN_MANAGE_USERS, self.user_id
+        )
+
+        response = self.request(
+            "user.create",
+            {
+                "username": "username",
+                "organisation_management_level": OrganisationManagementLevel.CAN_MANAGE_ORGANISATION,
+            },
+        )
+        self.assert_status_code(response, 403)
+        self.assertIn(
+            "Your Organisation Management Level is not high enough to set a Level of can_manage_organisation!",
+            response.json["message"],
+        )
+
+    def test_create_permission_set_1(self) -> None:
+        """ Rights for all field groups, but missing rights for meeting/4 """
+        self.create_meeting()
+        self.create_meeting(base=4)
+        self.user_id = self.create_user(
+            "test", group_ids=[2]
+        )  # admin-group of meeting/1
+        self.login(self.user_id)
+        self.set_management_level(
+            OrganisationManagementLevel.CAN_MANAGE_USERS, self.user_id
+        )
+        response = self.request(
+            "user.create",
+            {
+                # Group A
+                "username": "username",
+                "default_vote_weight": "1.700000",
+                "organisation_management_level": "can_manage_users",
+                "guest_meeting_ids": [1, 4],
+                "committee_as_member_ids": [78],
+                "committee_as_manager_ids": [78],
+                # Group B
+                "vote_delegations_$_from_ids": {1: [222]},
+                "comment_$": {1: "comment<iframe></iframe>"},
+                "number_$": {4: "number"},
+                "structure_level_$": {1: "level_1", 4: "level_2"},
+                "about_me_$": {1: "<p>about</p><iframe></iframe>"},
+                "vote_weight_$": {1: "1.000000", 4: "2.333333"},
+                # Group C
+                "group_$_ids": {1: [1]},
+            },
+        )
+
+        self.assert_status_code(response, 403)
+        self.assertIn("You do not belong to meeting 4", response.json["message"])
```

Comment:
so, `number_$` is the conflicting field? Maybe add it to the message (same as above).

Reply:
Done